script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -621,7 +622,7 @@ But it's abundantly clear that this man is anything but.
 [sha:a335a764e74c9d130c6a03319a42c5a7ce147101]{
 -- Page 517, Offset 21065.
 -- 「“―――ジャマ、ヲ”」
-"―――Don't get in my..."
+"Don't... get in my..."
```

Comment:
Should have some effects?
spaced this a bit (github might not display)

>d  o n't      ge t   i n    m y...

maybe italics too?